editoast/src/models/study.rs

```diff
@@ -130,12 +130,15 @@ impl List<(i64, Ordering)> for StudyWithScenarios {
     ) -> Result<PaginatedResponse<Self>> {
         let project_id = params.0;
         let ordering = params.1.to_sql();
-        sql_query(format!("SELECT t.*, COUNT(scenario.*) as scenarios_count FROM osrd_infra_study t
-            LEFT JOIN osrd_infra_scenario scenario ON scenario.study_id = t.id WHERE t.project_id = $1
-            GROUP BY t.id ORDER BY {ordering} "))
-            .bind::<BigInt, _>(project_id)
-            .paginate(page, page_size)
-            .load_and_count(conn).await
+        sql_query(format!(
+            "SELECT t.*, COUNT(scenario.*) as scenarios_count FROM study t
+            LEFT JOIN scenario scenario ON scenario.study_id = t.id WHERE t.project_id = $1
```

Comment:
```suggestion
            LEFT JOIN scenario ON scenario.study_id = t.id WHERE t.project_id = $1
```
Also I find that this `t` isn't a good choice